### DISCUSSION TITLE
How to get selected cells if the user selects multiple by using Shift + arrows?

### DISCUSSION THREAD

**Discussion by shelpuk** [bosskmk/pluto_grid#758] [Q&A]
When the user selects a single cell, I can access it with `stateManager.currentCell`. But how do I get selected cells if the user selects multiple by using Shift + arrows?

**Comment by bosskmk** [comment]
https://github.com/bosskmk/pluto_grid/issues/768

I've posted an answer to the above issue.
Please take a look and let me know in the comments if there is something missing.
thank you
